packages/gatsby-theme-api-docs/src/utils/ctp-urn.js

```diff
@@ -46,12 +54,15 @@ export const parseTypeURN = (urn) => {
   return false;
 };
 
-export const generateEndpointURN = ({
-  apiKey = '',
-  path = '',
-  method = '',
-}) => {
-  return `${prefix}:${encodeURNComponent(apiKey)}:endpoint:${encodeURNComponent(
+export const generateEndpointURN = (
+  { apiKey = '', path = '', method = '' },
+  isDom
+) => {
+  if (isDom) {
```

Comment:
I had to add this param to avoid escaping the actual URN identifier in the DOM, but still escape it for the URL.